catalog.tests.bom

```diff
@@ -0,0 +1,179 @@
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  id: brooklyn-mysql-node-tests
+  version: 0.10.0-SNAPSHOT
+  itemType: template
+  iconUrl: http://www-jp.mysql.com/common/logos/mysql-logo.svg
+  name: MySQL Node tests
+  description: Test that MySql Node is running correctly
+  license_code: Apache-2.0
+  license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+  item:
+    services:
+    - type: org.apache.brooklyn.test.framework.TestCase
+      name: MySQL Node tests
+      targetId: mysqlnode
+      brooklyn.children:
+      - type: brooklyn-mysql-node
+        id: mysqlnode
+        brooklyn.config:
+          datastore.creation.script.contents: |
+            CREATE DATABASE temporary;
+            USE temporary;
+            CREATE TABLE users (name VARCHAR(20), owner VARCHAR(20), creation_date DATE, updated_date DATE);
+            INSERT INTO users VALUES ('mysql', 'mysql', CURDATE(), CURDATE())
+          mysql.port: 6789
+
+      # Sensors tests
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: Sensors Tests
+        brooklyn.children:
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check MySQL server is up
+          sensor: service.isUp
+          timeout: 10m
+          assert:
+          - equals: true
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check MySQL port is setup correctly
+          sensor: mysql.port
+          timeout: 1m
+          assert:
+          - equals: 6789
+        - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
+          name: Check MySQL creating script has been executed
+          runDir: $brooklyn:entity("mysqlnode").attributeWhenReady("run.dir")
+          command: $brooklyn:formatString("%s/bin/mysql --defaults-file=mymysql.cnf -N -e 'SELECT count(*) FROM users' temporary", $brooklyn:entity("mysqlnode").attributeWhenReady("expandedinstall.dir"))
+          assertStatus:
+            equals: 0
+          assertOut:
+            equals: "1"
+          assertErr:
+            isEmpty: true
+
+      # Effector: executeScript Tests
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: "Effector: executeScript Tests"
+        brooklyn.children:
+        - type: org.apache.brooklyn.test.framework.TestEffector
+          name: Invoke executeScript Effector
+          effector: executeScript
+          params:
+            commands: |
+              USE temporary;
+              INSERT INTO users VALUES ('root', 'root', CURDATE(), CURDATE());
+              SELECT count(*) from users;
+          timeout: 1m
+          assertStatus:
+            equals: 0
+          assertOut:
+            matches: count\(\*\).*2
+          assertErr:
```

Comment:
Docs state that `assert` config is only available on `TestEffector`. I think these three are just nops.